### PR TITLE
python3-gitlab: update to 3.8.1.

### PR DESCRIPTION
**Pull request by TheGejr**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl (crossbuilds)
  - [x] aarch64 (crossbuilds)
  - [x] armv7l-musl (crossbuilds)
  - [x] armv7l (crossbuilds)
  - [x] i686 (crossbuilds)
  - [x] x86_64-musl (crossbuilds)

### CODE REVIEW

File: srcpkgs/python3-gitlab/template [hunk 1]
```diff
@@ -13,7 +13,7 @@ license="LGPL-3.0-or-later"
 homepage="https://github.com/python-gitlab/python-gitlab"
 changelog="https://raw.githubusercontent.com/python-gitlab/python-gitlab/master/ChangeLog.rst"
```
Comment (paper42) [inline srcpkgs/python3-gitlab/template:14]:
```suggestion
changelog="https://raw.githubusercontent.com/python-gitlab/python-gitlab/main/CHANGELOG.md"
```
the old URL returns 404

Comment (TheGejr) [reply srcpkgs/python3-gitlab/template]: Fixed now

**Comment by classabbyamp** [comment]
3.8.1 is available

File: srcpkgs/python3-gitlab/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/python3-gitlab/template:14]: still shows the old value for me

**Pull request closed without merging**